### DISCUSSION TITLE
Create external libray

### DISCUSSION THREAD

**Discussion by jcharcosset** [tailwindlabs/tailwindcss#7230] [Help]
Hi

I would like create library using Tailwind and [sfc-rollup](https://github.com/team-innovation/vue-sfc-rollup). I use vue 2.6, rollup 2.60, tailwind 3.0.

From my library, Tailwind works : 
![image](https://user-images.githubusercontent.com/36884315/151314102-c0b97213-7a75-4cbd-a783-b7a33cb2f3d7.png)

But, from my project, it doesn't work : 
![image](https://user-images.githubusercontent.com/36884315/151314181-c221598c-cf04-4503-a5f7-42f7e94b68aa.png)

I don't know where problem is located. Can you help me please ? I share code necessary if needed.

Thank you

**Comment by k3RetailDaveIzon** [comment]
Did you find an answer to this?
